types/react-relay/test/react-relay-tests.tsx

```diff
@@ -372,7 +383,7 @@ export const mutation = graphql`
 export const optimisticResponse = {
     markReadNotification: {
         notification: {
-            seenState: "SEEN",
+            seenState: "SEEN" as "SEEN",
```

Comment:
This is because normally TS will infer the type of this property to be `seenState: string`, which is too broad to satisfy the enum. Not sure if there’s a better way to do this 🤷‍♂️